frontend/src/textToSpeech/textToSpeech.js

```diff
@@ -0,0 +1,380 @@
+                        </div>
+                    </div>
+                    <div className='row answer'>
+                        <div className="col-1">
+                            <div className="play audio"
+                                onClick={this.playAudio}>
+                                {playButton('pink')}
+                            </div>
+                            <div className="play-button-buffer">
+                                {null}
+                            </div>
+                        </div>
+                        <div className='col-8 box-answer'>
+                            <h2 className='grade-text para'>Here is your grade:</h2>
+                            {
+                                this.state.graded
+                                    ? this.giveGrade()
+                                    : <p>You have not submitted yet</p>
+                            }
+                            {
+                                this.state.showAnswer
+                                    ? (
+                                        <div>
+                                            <br/>
+                                            <h2 className='para'>This is the correct answer:</h2>
+                                            <p>{this.getCurrentSentence()}</p>
+                                        </div>
+                                    )
+                                    : null
+                            }
+                        </div>
+                        <div className='col-3'>
+                            <div className='box-missing-words'>
+                                <h2 className='para'>Missing Words:</h2>
+                                {
+                                    this.state.graded
+                                        ? (
+                                            <div className='row'>
+                                                <div className='col missing-words-list'>
+                                                    {missingWord.slice(0,
+                                                        Math.round(missingWord.length / 2))}
+                                                </div>
+                                                <div className='col missing-words-list'>{
+                                                    missingWord.slice(Math.round(missingWord.length
+                                                        / 2), missingWord.length)}
+                                                </div>
+                                            </div>
+                                        )
+                                        : null
+                                }
+                            </div>
+                        </div>
+                    </div>
+                    <div className='row'>
+                        <div className="col box">
+                            <form id='input-form'>
```

Comment:
This form might need an `onSubmit={(event) => event.preventDefault()}` or something like that. Without it, I assume that we'll leave in default form behavior for the enter key press (i.e., it'll submit the form with no specified action and so reload the page).